components/raftstore-v2/src/operation/pd.rs

```diff
@@ -53,7 +53,8 @@ impl Store {
 
         // stats.set_query_stats(query_stats);
 
-        let task = pd::Task::StoreHeartbeat { stats };
+        let snap_size = ctx.snap_mgr.total_snap_size().unwrap();
```

Comment:
`total_snap_size` is heavy. Clone the `snap_mgr` to pd runner instead.

Reply:
fixed